app-backend/db/src/main/scala/ProjectDao.scala

```diff
@@ -467,4 +499,26 @@ object ProjectDao
       authProject <- authorized(user, ObjectType.Project, projectId, actionType)
       layerExist <- ProjectLayerDao.layerIsInProject(layerId, projectId)
     } yield { authProject && layerExist }
+
+  def removeLayerOverview(projectLayerId: UUID, locUrl: String): Unit = {
```

Comment:
This did not appear to trigger when I removed the last scene from the project layer

Reply:
I also didn't get any logging when I deleted the layer :thinking:

Reply:
Could wrap this in an IO to be more direct about effects -- since the effect type required might vary, you might get some mileage out of [`LiftIO`](https://typelevel.org/cats-effect/typeclasses/liftio.html)

Reply:
For deleting the last scene from a layer, it also logs for me:
```sql
rasterfoundry=# select overviews_location from project_layers where id = 'da0ccc38-cf30-4291-8e57-9b03d96d5f51';
 overviews_location 
--------------------
 s3://some/thing
```
```bash
api-server_1              | 2019-05-10T22:53:07.564834989Z 2019-05-10 22:53:07 INFO  ProjectDao$:506 - No scenes left in project layer da0ccc38-cf30-4291-8e57-9b03d96d5f51 with overview location s3://some/thing
api-server_1              | 2019-05-10T22:53:08.102497399Z 2019-05-10 22:53:08 INFO  ProjectDao$:521 - Not Found overview: s3://some/thing for layer da0ccc38-cf30-4291-8e57-9b03d96d5f51
nginx-api_1               | 2019-05-10T22:53:08.203771329Z 10.0.2.2 - - "DELETE /api/projects/965b81ea-ab19-437d-a90f-6a14dccc106e/scenes/ HTTP/1.1" 204 0 - "http://localhost:9091/projects/edit/965b81ea-ab19-437d-a90f-6a14dccc106e/scenes?page=1" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.131 Safari/537.36" 3.191 3.187
```